game/src/challenges/cutscene.rs

```diff
@@ -145,8 +145,8 @@ fn make_panel(
     make_task: &Box<dyn Fn(&mut EventCtx) -> Widget>,
     idx: usize,
 ) -> Panel {
-    let prev_builder = ButtonStyle::outline_dark_fg()
-        .plain_icon("system/assets/tools/circled_prev.svg")
+    let prev_builder = ButtonStyle::plain_dark_fg()
```

Comment:
Previously we didn't have a separate ButtonStyle for plain - it was just the "outline" style without the outline applied. 

But it was the StyledButtons trait which managed this. Since I wanted to get rid of the StyledButtons trait, I introduced some new flavors of "plain" ButtonStyle's. 

I'm playing a bit of whack-a-mole here, but I think this approach helps more than it hurts.